.github/workflows/ci.yml

```
@@ -0,0 +1,27 @@
+name: Continuous Integration
+
+on:
+  push:
+  pull_request:
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+
+    steps:
+      - uses: actions/checkout@v2
```

For example, why are we using the Checkout v2 version? You could simply do like this: 

```suggestion
      - uses: actions/checkout@692973e3d937129bcbf40652eb9f2f61becf3332 #v4.1.7
```